Rollbar/RollbarQueueController.cs

```diff
@@ -340,5 +352,86 @@ private void OnRollbarEvent(RollbarEventArgs e)
             e.Logger.OnRollbarEvent(e);
         }
 
+        /// <summary>
+        /// Gets the total payload count across all the queues.
+        /// </summary>
+        /// <returns></returns>
+        public int GetTotalPayloadCount()
+        {
+            lock (this._syncLock)
```

Comment:
It seems like we are using two locks for these queues, shouldn't we only need one point of synchronization?

Reply:
which one is the second lock. The this._syncLock is the only one . Did I miss something?